modules/SofaConstraint/LCPConstraintSolver.cpp

```diff
@@ -1026,16 +1026,12 @@ int LCPConstraintSolver::nlcp_gaussseidel_unbuilt(double *dfree, double *f, std:
         std::list<unsigned int>::iterator it_c ;
         error =0;
 
-        for (it_c = contact_sequence.begin(); it_c != contact_sequence.end() ; ++it_c )
+        //constraints are treated 3x3 (friction contact)
+        for (it_c = contact_sequence.begin(); it_c != contact_sequence.end() ; ++(++(++it_c)) )
```

Comment:
wtf? ++(++(++ ?
it_c += 3 ?

Reply:
Wanted to be sure there would be not post-increment problem.

Reply:
:)